client/main.coffee

```diff
@@ -1275,6 +1290,24 @@ class Board
     @transform.y += @bbox.height/2 * (1/newScale - 1/@transform.scale)
     @transform.scale = newScale
     @retransform()
+  zoomToFit: (min, max) ->
+    # Change the bounds so as to fit the rect from (min, max) on screen
+    width = max.x - min.x
+    height = max.y - min.y
+    midx = 0.5 * (min.x + max.x)
+    midy = 0.5 * (min.y + max.y)
```

Comment:
I originally thought we should be able to avoid finding the center. But then I realized we need to find at least one of the centers, because the zoom will be constrained in one dimension but not the other. Makes sense.